### DISCUSSION TITLE
How to autosave form one input at a time if filled correctly?

### DISCUSSION THREAD

**Discussion by waldothedeveloper** [edmundhung/conform#359] [Q&A]
I was reading this post from Jacob Paris: https://www.jacobparis.com/content/remix-form-autosave that uses

```javascript
const fetcher = useFetcher()
``` 
 and

```javascript
    <fetcher.Form method="POST" {...form.props}>
      <input {...conform.input(fields.email)} />
      <input {...conform.input(fields.name)} />
      <button type="submit">
        {fetcher.state === "submitting"
          ? "Saving…"
          : "Save"}
      </button>
    </fetcher.Form>
``` 

to autosave a form with some other stuff. I might be wrong, but I think that might be achieved by using `shouldValidate: 'onBlur',
		shouldRevalidate: 'onBlur',`  from `useForm`.

My issue is that Conform validates all fileds by default, (I'm also doing client validation) so assuming I have multiple inputs or a complex form how can I achieve saving each input as the user finishes typing and that field has no errors?

**Comment by waldothedeveloper** [comment]
I was able to chat with Jacob yesterday about this, and now I have a clear understanding on what to do to achieve this :)

**Reply by kirillsbarsukov** [reply]
@waldothedeveloper, could you please share the result of your conversation?
